moabb/datasets/download.py

```diff
@@ -82,7 +79,7 @@ def data_dl(url, sign, path=None, force_update=False, update_path=True, verbose=
     sign : str
         Signifier of dataset
     path : None | str
-        Location of where to look for the BNCI data storing location.
+        Location of where to look for the data storing location.
         If None, the environment variable or config parameter
         ``MNE_DATASETS_(signifier)_PATH`` is used. If it doesn't exist, the
         "~/mne_data" directory is used. If the dataset
```

Comment:
You could remove the update_path argument from the function in data_dl. 

Usually when you change the signature of a function (the name or number of arguments), this is an API break and require a major update, like going from MOABB v0.3.0 to 1.0.0 

Here, it is a special case, as we don't need to consider that API compatibility is broken as I just introduced this function, it is still in "dev branch". 

For the `data_path` function just above, we need to keep the `update_path`argument, as the code is released and stable and someone may have use this function. We could not break it without warning.

Reply:
Has data_dl only been used in BNCI and SSVEPEXO?